apps/src/craft/aquatic/craft.js

```diff
@@ -533,6 +534,10 @@ Craft.executeUserCode = function () {
     },
   };
 
+  const isWalkable = block => {
+    return block && block.isWalkable;
+  };
```

Comment:
should this change be replicated in the other `craft.js`s?

Reply:
This is the first year we've used `isWalkable`.